packs/st2cd/actions/workflows/mistral_itests.yaml

```diff
@@ -0,0 +1,119 @@
+---
+  chain:
+    -
+      name: "get_build_server"
+      ref: "linux.dig"
+      params:
+        hostname: "st2build.service.consul"
+        rand: true
+        count: 1
+      on-success: "clone_st2_repo"
+    -
+      name: "clone_st2_repo"
+      ref: "st2cd.git_clone"
+      params:
+        hosts: "{{get_build_server.result[0]}}"
+        repo: "{{st2_repo}}"
+        branch: "{{st2_branch}}"
+        target: "{{st2_repo_target}}"
+      on-success: "clone_mistral_repo"
+    -
+      name: "clone_mistral_repo"
+      ref: "st2cd.git_clone"
+      params:
+        hosts: "{{get_build_server.result[0]}}"
+        repo: "{{st2_mistral_fork}}"
+        branch: "{{st2_mistral_fork_branch}}"
+        target: "{{mistral_repo_target}}"
+      on-success: "clone_st2mistral_repo"
+    -
+      name: "clone_st2mistral_repo"
+      ref: "st2cd.git_clone"
+      params:
+        hosts: "{{get_build_server.result[0]}}"
+        repo: "{{st2mistral_repo}}"
+        branch: "{{st2mistral_branch}}"
+        target: "{{st2mistral_repo_target}}"
+      on-success: "clone_mistral_client_repo"
+    -
+      name: "clone_mistral_client_repo"
+      ref: "st2cd.git_clone"
+      params:
+        hosts: "{{get_build_server.result[0]}}"
+        repo: "{{mistral_client_repo}}"
+        branch: "{{mistral_client_branch}}"
+        target: "{{mistral_client_repo_target}}"
+      on-success: "install_mistral_st2action"
+    -
+      name: "install_mistral_st2action"
+      ref: "st2cd.install_st2action"
+      params:
+        hosts: "{{get_build_server.result[0]}}"
+        repo: "{{st2mistral_repo}}"
+        branch: "{{st2mistral_branch}}"
+      on-success: "install_mistral_st2action"
+    -
+      name: "install_mistral_client"
+      ref: "st2cd.install_mistral_client"
+      params:
+        hosts: "{{get_build_server.result[0]}}"
+        repo: "{{mistral_client_repo}}"
+        branch: "{{mistral_client_branch}}"
+      on-success: "install_mistral_st2action"
+    -
+      name: "install_mistral"
```

Comment:
NOTE: Apparently, st2mistral needs to be installed inside the virtualenv of mistral.